[debug] [2020-04-02T19:02:41.020Z] ----------------------------------------------------------------------
[debug] [2020-04-02T19:02:41.022Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Bg\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-02T19:02:41.022Z] CLI Version:   8.0.0
[debug] [2020-04-02T19:02:41.022Z] Platform:      win32
[debug] [2020-04-02T19:02:41.023Z] Node Version:  v10.13.0
[debug] [2020-04-02T19:02:41.023Z] Time:          Thu Apr 02 2020 21:02:41 GMT+0200 (Central European Summer Time)
[debug] [2020-04-02T19:02:41.023Z] ----------------------------------------------------------------------
[debug] [2020-04-02T19:02:41.023Z] 
[debug] [2020-04-02T19:02:41.038Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-02T19:02:41.069Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\Bg\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
